doc/src/conf.py

```diff
@@ -30,6 +30,14 @@
               'sphinx.ext.mathjax', 'numpydoc', 'sympylive',
               'sphinx.ext.graphviz', 'matplotlib.sphinxext.plot_directive']
 
+redirects = {
+    "install.rst": "getting_started/install.rst",
+    "aboutus.rst": "getting_started/aboutus.rst",
+    "wiki.rst": "getting_started/wiki.rst",
+    "citing.rst": "getting_started/citing.rst",
+    "outreach.rst": "getting_started/outreach.rst",
```

Comment:
This page is also not so useful. I'd delete and move any useful content to the sympy website.